.travis.yml

```diff
@@ -154,7 +150,6 @@ matrix:
         postgresql: "9.4"
       env:
         - PG_VERSION=9.4
-        - MCENTRAL=Y
```

Comment:
I think this `MCENTRAL=Y` was there for a good reason: deploy jre7-SNAPSHOT to Central.